airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionStatusTab.tsx

```diff
@@ -177,7 +177,7 @@ export const ConnectionStatusTab: React.FC = () => {
                       onClick={onSyncNowButtonClick}
                       icon={
                         <div className={styles.iconRotate}>
-                          <RotateIcon />
+                          <RotateIcon height={styles.syncIconHeight} width={styles.syncIconHeight} />
```

Comment:
This change and the SCSS changes for this component are to fix the styling of the Sync button on the connection page.

Before:
<img width="121" alt="Screen Shot 2022-10-25 at 2 47 18 PM" src="https://user-images.githubusercontent.com/22731524/197888462-9fba9b93-8b8d-4902-b0a8-708f691398e0.png">

After:
<img width="121" alt="Screen Shot 2022-10-25 at 2 47 31 PM" src="https://user-images.githubusercontent.com/22731524/197888475-1a3c60ad-f38b-4f62-929a-bfffa4aa7b13.png">